kubernetes.js

```diff
@@ -434,6 +508,24 @@ const createProject = async (project, options) => {
             }
         }
     }
+    if (this._customHostname?.enabled) {
+        const customHostname = await project.getSetting('customHostname')
+        if (customHostname) {
+            const customHostnameIngress = await createCustomIngress(project, customHostname, options)
+            try {
+                await this._k8sNetApi.createNamespacedIngress(namespace, customHostnameIngress)
+            } catch (err) {
+                if (err.statusCode === 409) {
+                    this._app.log.warn(`[k8s] Custom Hostname Ingress for project ${project.id} already exists, proceeding...`)
```

Comment:
```suggestion
                    this._app.log.warn(`[k8s] Custom Hostname Ingress for instance ${project.id} already exists, proceeding...`)
```

Can you update all the other log messages similarly?